workspace/TS100/src/main.cpp

```diff
@@ -87,6 +89,21 @@ int main(void) {
 	while (1) {
 	}
 }
+
+void debugNumber(int32_t val) {
```

Comment:
Any chance you could move this to the OLED class to keep things neater ?

Reply:
Absolutely! I just wasn't sure where to put it...